src/standalone/standalone.js

```diff
@@ -10,17 +10,16 @@ function parseURLParameters() {
     var URL;
     if (window.location.hash.length > 0) {
         // Prefered method since parameters aren't sent to server
-        URL = [window.location.hash.slice(1)];
+        URL = window.location.hash.slice(1);
     } else {
-        URL = decodeURI(window.location.href).split('?');
```

Comment:
If we decode the whole URI before extracting the parameters we risk looping through parameters that are from query string variables such as `panorama`. And we will also not get the right URL for `panorama` or other parameter that is a URL.